docker-compose.yml

```diff
@@ -0,0 +1,76 @@
+# This file can be used to create a development environment for Wordpress
+# using Docker.
+#
+#
+# To set up
+# =========
+#
+# Download a recent backup of the database from Azure:
+#

```

Comment:
You might be able to do some fancy things with Azure CLI https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest to download the most recent backup image. Or we could have a job that puts it in a file share on a regular basis and then the docker image could pull from that.

Reply:
That's a good idea.  It might be nice to have the backup available locally so we can pull it as needed.